packages/ui/components/Accordion/index.tsx

```diff
@@ -0,0 +1,180 @@
+'use client';
+
+import React, { useCallback, useMemo, useState } from 'react';
+import RichText from '@mono/ui/components/primitives/RichText';
+import Icon from '@mono/ui/components/RenderIcon';
+import s, { css } from '@refract-ui/sc';
+
+const ContentStyles = s.div`
+  padding-bottom: 1.125rem;
+`;
+
+const Content = s.div`
+  overflow: hidden;
+  height: 0;
+  display: none;
+
+  &.open {
+    height: auto;
+    display: block;
+  }
+`;
+
+const IconContainer = s.div`
+  display: flex;
+  align-items: center;
+  column-gap: 0.75rem;
+`;
+
+const Text = s.span`
+  display: block;
+
+  ${({ theme: { box, allColors } }) => css`
+    ${box.t('h3')};
+    color: ${allColors.plainFg};
+  `}
+`;
+
+const hoverStyles = css`
+  ${({ theme: { allColors } }) => css`
+    &:hover {
+      outline: none;
+
+      svg {
+        color: ${allColors.plainHover};
+      }
+
+      ${Text} {
+        color: ${allColors.plainHover};
+        text-decoration: underline;
+      }
+    }
+  `}
+`;
+
+const TextButton = s.button`
+  all: unset;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  cursor: pointer;
+  column-gap: 2rem;
+  width: 100%;
+  padding: 1.125rem 0;
+
+  &:hover {
+    background: transparent;
+    box-shadow: none;
+  }
+
+  ${hoverStyles};
+`;
+
+const InnerList = s.li`
+  list-style: none;
+  margin: 0;
+  cursor: pointer;
+  overflow: hidden;
+
+  ${({ theme: { allColors } }) => css`
+    border-top: 1px solid ${allColors.plainFg};
+
+    svg {
+      color: ${allColors.plainFg};
+    }
+
+    ${hoverStyles};
+  `}
+`;
+
+const List = s.ul`
+  margin: 0;
+  padding: 0;
+`;
+
+export type AccordionProps = {
+  title?: string;
+  items?: {
```

Comment:
I feel like we should be able to use a payload-type for this. We might need to come back after the `FAQBlock` is updated to refactor this type. Lets leave a comment in the code about this. I also update the FAQBlock ticket